.artifact/tree.json

```diff
@@ -168,72 +168,6 @@
     },
     "example_cartographer_outputs": {
       "viam-office-02-22-1": {
-        "image_map": {
```

Comment:
where are these used in viam cartographer?

Reply:
`viam-cartographer` and `viam-orbslam` will soon have the integration tests run through them instead of rdk.